manifests/server.pp

```diff
@@ -161,6 +161,7 @@
     conn_keep_alive     => $read_conn_keep_alive,
     conn_lifetime       => $read_conn_lifetime,
     confdir             => $confdir,
+    require             => Package[$puppetdb_package],
```

Comment:
If this require is needed here, will it be needed in the other config file resources as well?